<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>standardization_window</class>
 <widget class="QWidget" name="standardization_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标定</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>旋转标定</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>位移标定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="sw_stan">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>第一步：请在主界面设置模板，请确认模板工件中心</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_standardization">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>第三步：点击&quot;标定&quot;按钮获取角度与像素点之间的转换系数</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pb_isTrue">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>第二步：请控制PLC携带工件旋转1°</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pb_standardization">
         <property name="text">
          <string>标定</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>第四步：点击&quot;确定&quot;按钮获取确定设置标定系数</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>第一步：请在主界面设置模板，请确认模板工件中心</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>第二步：请控制PLC在轴向相对模板位置位移一定距离</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="text">
          <string>第三步：点击“第一个点”按钮</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="text">
          <string>第四步：请控制PLC在轴向相对模板位置位移一定距离</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="text">
          <string>第五步：点击“第二个点”按钮</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="text">
          <string>第六步：点击&quot;确定&quot;按钮获取确定设置标定系数</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pb_first">
         <property name="text">
          <string>第一个点</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pb_second">
         <property name="text">
          <string>第二个点</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pb_true">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="le_firstPoint"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="le_second_Point"/>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="le_true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
